<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
border:5px solid white;
background-color:qlineargradient(spread:pad, x1:0.164773, y1:0.222, x2:1, y2:1, stop:0.0170455 rgba(8, 86, 0, 255), stop:1 rgba(0, 0, 0, 255));}

</string>
    </property>
    <widget class="QTextEdit" name="screen">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>8</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 7.5pt &quot;DejaVu Sans Mono&quot;;
background-color:rgba(0, 0, 0,0.9);
color:white;
border-radius:20px;
padding-left:20;
padding-right:20;
padding-top:10</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>0</y>
       <width>21</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:7px solid white;
background-color:rgba(255, 255, 255, 0.3);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>99</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);
hover{color:black};</string>
      </property>
      <property name="text">
       <string>Add Item</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_14">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>20</y>
       <width>641</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>120</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="database_info">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>398</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 0);
padding-left:5;
background-color:rgba(0, 0, 0,0.5);
font: 63 15pt &quot;Bahnschrift&quot;;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>15</number>
      </property>
      <property name="text">
       <string>Unamed Database</string>
      </property>
     </widget>
     <widget class="QLabel" name="database_info_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Bahnschrift Condensed&quot;;
color:rgb(255, 255, 0);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>15</number>
      </property>
      <property name="text">
       <string>Status:Active</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dashboard_icon">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;PMingLiU-ExtB&quot;;
color:rgb(255, 255, 255);
border:1px solid white;
border-radius:15;</string>
     </property>
     <property name="text">
      <string>Inventory Manager</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Downloads/677777.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>233</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>117</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="delet">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>96</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);
hover{color:black};</string>
      </property>
      <property name="text">
       <string>Delete Item</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>153</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="Iout">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);
hover{color:black};</string>
      </property>
      <property name="text">
       <string>Item Out</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>193</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="Iin">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>102</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);
hover{color:black};</string>
      </property>
      <property name="text">
       <string>Item In</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>452</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="c_password">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>1</y>
        <width>96</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Change password</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>487</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="logout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>414</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="report">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>1</y>
        <width>96</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Create Report</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>271</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255,0.1);
border-radius:20;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="find">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>-1</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
font: 75 10pt &quot;Noto Sans Cond&quot;;
color:rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>112</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/delete.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>81</y>
       <width>17</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/add.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>153</y>
       <width>20</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/out.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>193</y>
       <width>17</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/in.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>234</y>
       <width>18</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/refresh.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>272</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/find.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>453</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/icons/Change password.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>frame_4</zorder>
    <zorder>frame_5</zorder>
    <zorder>frame_6</zorder>
    <zorder>frame_8</zorder>
    <zorder>frame_7</zorder>
    <zorder>label_2</zorder>
    <zorder>screen</zorder>
    <zorder>line_2</zorder>
    <zorder>frame_2</zorder>
    <zorder>frame_14</zorder>
    <zorder>label</zorder>
    <zorder>frame_3</zorder>
    <zorder>frame_9</zorder>
    <zorder>frame_10</zorder>
    <zorder>label_5</zorder>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
